client/wfm/src/step/Step.ts

```diff
@@ -2,13 +2,17 @@
  * Definition for a {@link Task} containing its type identifier and runtime options for customizing behavior.
  * Should be contained in a {@link Process} definition
  */
-export interface TaskDefinition {
+export interface Step {
   /**
    * Code that matches the identifier in {@link TaskHandler}
    */
   code: string;
+  name: string;
   /**
-   * Options object that act as runtime parameters for its equivalent {@link Task}
+   * Options object that act as runtime parameters for its equivalent {@link Step}
    */
-  options?: object;
+  templates?: {
```

Comment:
+1 for having this still in the datastore for the moment until we figure out how to migrate this to templates being part of the step. Checking code usages suggest that this may be a non trivial task:
https://github.com/feedhenry-raincatcher/raincatcher-angularjs/search?utf8=%E2%9C%93&q=templates.form&type=